components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -152,6 +156,19 @@ export default class CreateWorkspace extends React.Component<CreateWorkspaceProp
   }
 }
 
+function LimitReachedModal(p: { children: React.ReactNode }) {
+  return <Modal visible={true} closeable={false} onClose={()=>{}}>
+    <h3>Limit Reached</h3>
+    <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-2">
+      {p.children}
+    </div>
+    <div className="flex justify-end mt-6">
+      <a href={gitpodHostUrl.asDashboard().toString()}><button className="secondary">Go back to dashboard</button></a>
```

Comment:
**nitpick**: Small rephrase as users may reach this without going through the dashboard, right?

```suggestion
      <a href={gitpodHostUrl.asDashboard().toString()}><button className="secondary">Go to Dashboard</button></a>
```